packages/controller/buidler.config.4.js

```diff
@@ -0,0 +1,14 @@
+// This buidler configuration is only used to compile ANTMock.sol for testing,
+// as it relies on an older solc version (0.4.8)
+module.exports = {
+  solc: {
+    version: "0.4.8",
+    optimizer: {
+      enabled: true,
+      runs: 200,
```

Comment:
Any reason for using only 200 runs? Is it to follow the previous runs configuration?

Reply:
Let's actually raise it as much as possible, any optimization on `onTransfer()` is going to save so much gas in the future

Reply:
Yes, it's 200 because the mainnet ANT contract is deployed with 200 runs.

Note that the controller contract is compiled on 0.5.17 with 10k runs (although obviously it barely makes a difference here).